src/RnExecutorch.ts

```diff
@@ -1,15 +1,12 @@
 import { useCallback, useEffect, useRef, useState } from 'react';
-import { Image } from 'react-native';
-import RnExecutorch, {
-  subscribeToDownloadProgress,
-  subscribeToTokenGenerated,
-} from './RnExecutorchModule';
+import { EventSubscription, Image } from 'react-native';
+import { ResourceSource, Model } from './types';
+import RnExecutorch from './native/RnExecutorchModule';
 import {
-  DEFAULT_CONTEXT_WINDOW_LENGTH,
   DEFAULT_SYSTEM_PROMPT,
+  DEFAULT_CONTEXT_WINDOW_LENGTH,
   EOT_TOKEN,
-} from './constants';
-import { ResourceSource, Model } from './types';
+} from './constants/constants';
```

Comment:
also, maybe not for now, but the naming looks off (constants file inside constants dir) just a thought tho